test/e2e/suites/import-gitops-v3/import_gitops_v3_test.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/rancher/turtles/test/e2e/specs"
 )
 
-var _ = Describe("[Docker] [Kubeadm] - [management.cattle.io/v3] Create and delete CAPI cluster functionality should work with namespace auto-import", Label(e2e.ShortTestLabel), func() {
+var _ = Describe("[Docker] [RKE2] - [management.cattle.io/v3] Create and delete CAPI cluster functionality should work with namespace auto-import", Label(e2e.ShortTestLabel), func() {
```

Comment:
I know @Danil-Grigorev saw some memory limits issues on the github runner when using RKE2 in a short run

Reply:
It's probably what I've when testing this. Passed locally but failed remotely. Could this be affecting nightly tests too?